### DISCUSSION TITLE
eww

### DISCUSSION THREAD

**Discussion by s03i0path** [end-4/dots-hyprland#116] [Q&A]
hey man, why is my eww not working, i mean at all ... any suggestion?

**Comment by end-4** [comment]
either because
- the eww magic variable EWW_TIME doesn't work
- the sidebars are supposed to open with animations. i have scripts for each that open the window internally and change css margins to make it reveal. *this currently does not work, and i can't be bothered to find out why*, since nowi'm using AGS instead of EWW

**Reply by s03i0path** [reply]
even ags is not working...

**Comment by clsty** [comment]
EWW is not longer supported in favor of AGS. Closing.
